frontend/pages/hosts/details/DeviceUserPage/DeviceUserPage.tsx

```diff
@@ -150,7 +148,9 @@ const DeviceUserPage = ({
           // exit early because refectch is pending so we can avoid unecessary steps below
         }
       },
-      onError: (error) => handlePageError(error),
+      onError: (error) => {
```

Comment:
Should we remove this log?

Reply:
Yeah, I think so,  I'll stick to the pattern of omitting logging API errors